akka-persistence-typed/src/test/java/akka/persistence/typed/javadsl/PersistentActorJavaDslTest.java

```diff
@@ -452,13 +452,13 @@ public void tapPersistentActor() {
     TestProbe<Signal> signalProbe = testKit.createTestProbe();
     BehaviorInterceptor<Command, Command> tap = new BehaviorInterceptor<Command, Command>() {
       @Override
-      public Behavior<Command> aroundReceive(akka.actor.typed.ActorContext<Command> ctx, Command msg, ReceiveTarget<Command> target) {
+      public Behavior<Command> aroundReceive(akka.actor.typed.TypedActorContext<Command> ctx, Command msg, ReceiveTarget<Command> target) {
         interceptProbe.ref().tell(msg);
         return target.apply(ctx, msg);
       }
 
       @Override
-      public Behavior<Command> aroundSignal(akka.actor.typed.ActorContext<Command> ctx, Signal signal, SignalTarget<Command> target) {
+      public Behavior<Command> aroundSignal(akka.actor.typed.TypedActorContext<Command> ctx, Signal signal, SignalTarget<Command> target) {
```

Comment:
These two can now be imported.  The reason they're fully qualified here is the motivation for this PR.

Reply:
Good catch!